how-to/web-interop-support-context-and-intents/client/src/platform/layout/layout-override.ts

```diff
@@ -0,0 +1,258 @@
+				const platformLayoutSnapshot: PlatformLayoutSnapshot = {
+					layouts: layoutSnapshot.layouts,
+					layoutTitles: {},
+					layoutSelected: this._selectedLayout
+				};
+				for (const layout of this._layoutMapArray) {
+					if (layout.layoutTitle !== undefined) {
+						platformLayoutSnapshot.layoutTitles[layout.layoutName] = layout.layoutTitle;
+					}
+				}
+				return platformLayoutSnapshot;
+			}
+
+			/**
+			 * Override for applying multiple snapshots.
+			 * @param snapshot The layouts object containing the fixed set of available layouts.
+			 */
+			public async applyLayoutSnapshot(snapshot: WebLayoutSnapshot): Promise<void> {
+				console.log(`[Apply Layout] Does this exist? ${Boolean(this._layoutContainer)}`);
+				if (this._layoutContainer !== null && this._layoutContainer !== undefined) {
+					const platformLayoutSnapshot = snapshot as PlatformLayoutSnapshot;
+					for (const [key, value] of Object.entries(snapshot.layouts)) {
+						const layoutTitle =
+							platformLayoutSnapshot?.layoutTitles === undefined
+								? undefined
+								: platformLayoutSnapshot.layoutTitles[key];
+						this._layoutMapArray.push({
+							layoutName: key,
+							layoutTitle,
+							layout: value,
+							container: this._layoutContainer
+						});
+					}
+					setTimeout(async () => {
+						const entries = Object.entries(platformLayoutSnapshot.layouts);
+						let entryInstance = 0;
+						for (const entry of entries) {
+							entryInstance++;
+							const layoutName = entry[0];
+							await this.createLayout(
+								layoutName,
+								entry[1],
+								entryInstance,
+								entries.length,
+								platformLayoutSnapshot.layoutSelected
+							);
+						}
+					}, 1000);
+					console.log("[Apply Layout] Layouts loaded");
+					console.log(`[Apply Layout] Layouts are: ${JSON.stringify(this._layoutMapArray)}`);
+				}
+			}
+
+			/**
+			 * Shows the layout specified by the layoutName.
+			 * @param layoutName The name of the layout to show.
+			 * @param layoutName.layoutName The name of the layout to show.
+			 * @returns Promise<void>
+			 */
+			public async showLayout({ layoutName }: OpenFin.LayoutIdentity): Promise<void> {
+				const layoutContainers = document.querySelectorAll<HTMLElement>("div.layout-container");
+				for (const layoutContainer of layoutContainers) {
+					if (layoutContainer.id === layoutName) {
+						layoutContainer.style.display = "block";
+						this._selectedLayout = layoutName;
+					} else {
+						layoutContainer.style.display = "none";
+					}
+				}
+			}
+
+			/**
+			 * Remove Layout - You guessed it, it removes a layout from the existing array of layouts.
+			 * @param id The name of the layout you want removed.
+			 */
+			public async removeLayout(id: OpenFin.LayoutIdentity): Promise<void> {
+				const index = this._layoutMapArray.findIndex((x) => x.layoutName === id.layoutName);
+				console.log(`[LM Override] Removing Layout ${id.layoutName}`);
```

Comment:
nit: inconsistent log prefix, above you use `[Method Name]`

Reply:
Updated.